src/pages/single-cover/SingleCover.js

```diff
@@ -59,19 +60,45 @@ export default function SingleCover({ cover, setOpenModal }) {
               <li>
                 <p> Source</p>
                 <a href={cover?.fields?.Source} target="_blank">
-                  <h3 className={classes.truncate}>{cover?.fields?.Source}</h3>
+                  <h3 className={classes.truncate}>
```

Comment:
The logic here should transform the URL rather than be hardcoded e.g. www.boomplay.com/x -> boomplay.com